crypto/property/property.c

```diff
@@ -474,7 +474,8 @@ static void alg_do_each(ossl_uintmax_t idx, ALGORITHM *alg, void *arg)
     for (i = 0; i < end; i++) {
         IMPLEMENTATION *impl = sk_IMPLEMENTATION_value(alg->impls, i);
 
-        alg_do_one(alg, impl, data->fn, data->fnarg);
+        if (impl != NULL)
```

Comment:
I don't see how this can possibly be NULL.  The only place that adds to the stack ensures that NULL isn't added.

Reply:
I'd be suspecting memory corruption.  Have you run with an address sanitiser or valgrind?